control-plane/connect-inject/envoy_sidecar_test.go

```diff
@@ -280,6 +315,7 @@ func TestHandlerEnvoySidecar_EnvoyExtraArgs(t *testing.T) {
 			expectedContainerCommand: []string{
 				"envoy",
 				"--config-path", "/consul/connect-inject/envoy-bootstrap.yaml",
+				"--concurrency", "0",
```

Comment:
Non-blocking nit: Should these all be set to 2 since that is the default?

Reply:
An alternative approach would be for me to do something like this in every test:
```
	h := Handler{
				ImageConsul:    "hashicorp/consul:latest",
				ImageEnvoy:     "hashicorp/consul-k8s:latest",
				EnvoyExtraArgs: tc.envoyExtraArgs,
                                 EnvoyDefaultEnvoyConcurrency: 2,
			}
```
and then assert on it being "2" but its also kinda hacky, and I have a test already which validates being able to set the values and its defaults at the top of this test file.
Thoughts?